### ISSUE TITLE
Unknown option `--node-args' on pm2-dev

### ISSUE THREAD

**Issue by pedrobarros** [Unitech/pm2#2992]
**What's going wrong?**
I can't set a debug port on pm2-dev CLI. 

When running `pm2-dev start --node-args="--debug=7000" bin/www`, I get `error: unknown option '--node-args'`

Am I missing something? Is there another way to pass node args on pm2-dev or another way to explicitly set a debug port?

**How could we reproduce this issue?**
Running `pm2-dev start --node-args="--debug=7000" bin/www`

## Supporting information

```
PM2 version: 2.5.0
Node version: 6.9.2
On Mac.
```

**Comment by vmarchaud** [comment]
You can't use `--node-args` with `pm2-dev` you must use the `pm2` binary directly.

**Comment by Unitech** [comment]
Option added to pm2-dev

You can use it now:

```bash
$ npm install Unitech/pm2#development -g
$ pm2-dev app.js --node-args="--harmony"
```

**Comment by Unitech** [comment]
Published on PM2 2.6.0:

```bash
$ npm install pm2@latest -g
$ pm2 update
$ pm2-dev app.js --node-args="--harmony"
```
